keyboards/lily58/rules.mk

```diff
@@ -29,12 +37,7 @@ BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
 # Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
 SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
-
-CUSTOM_MATRIX = yes
-
-SRC += i2c.c
-SRC += serial.c
-SRC += ssd1306.c
+OLED_DRIVER_ENABLE=yes      # OLED display
 
 # A workaround until #7089 is merged.
```

Comment:
hey @yyc this workaround came in the rebase/merge and maybe is no longer required when using common split code?

Reply:
That may be, but `OLED_DRIVER` is much more robust than the `ssd1306.c` implementation.  Including support for PROGMEM, display rotation, scrolling, and raw rendering